inference/core/env.py

```diff
@@ -179,6 +182,20 @@ def required_providers_env(val):
 # Maximum number of detections, default is 300
 MAX_DETECTIONS = int(os.getenv("MAX_DETECTIONS", 300))
 
+# Loop interval for expiration of memory cache, default is 5
```

Comment:
Yes, seconds. My thought was that we want to be checking and cleaning up expired keys often so that we don't risk eating up all the memory for a highly utilized server.

Reply:
Sounds good. Small suggestion to update this and line 193 to `X seconds` for clarity.